content/doc/developer/plugin-development/index.adoc

```diff
@@ -2,30 +2,11 @@
 title: Plugin development
 summary: About Jenkins' internal extension functionality, modules, and plugins
 layout: developerchapter
-wip: true
 references:
 - url: https://github.com/jenkinsci/plugin-pom
   title: Plugin parent POM on GitHub
-- url: https://wiki.jenkins.io/display/JENKINS/Adding+tool+auto-installer
-  title: Adding tool auto-installer
-- url: https://wiki.jenkins.io/display/JENKINS/Choosing+Jenkins+version+to+build+against
-  title: Choosing Jenkins version to build against
-- url: https://wiki.jenkins.io/display/JENKINS/Dependencies+among+plugins
-  title: Dependencies among plugins
-- url: https://wiki.jenkins.io/display/JENKINS/Tips+for+optional+dependencies
-  title: Tips for optional dependencies
 - url: ../blueocean-plugin-development
   title: Making your plugin work with Blue Ocean
-- url: https://wiki.jenkins.io/display/JENKINS/Marking+a+new+plugin+version+as+incompatible+with+older+versions
-  title: Marking a new plugin version as incompatible with older versions
-- url: https://wiki.jenkins.io/display/JENKINS/Site+Creation+and+Deployment
-  title: Site Creation and Deployment
```

Comment:
Seems irrelevant these days? just use github docs?

Reply:
There are some plugins using GitHub pages, e.g. K8s Credentials. 
But I also do not see it as important.
My only suggestion is to comment out deprecated references instead of just removing them. It would be great to have TODOs so that we can cleanup Wiki later